tools/api/generate_go_protobuf.py

```diff
@@ -40,8 +40,8 @@ def generateProtobufs(output):
     # Example output directory:
     # go_out/envoy/config/bootstrap/v2
     rule_dir, proto = rule.decode()[len('@envoy_api//'):].rsplit(':', 1)
-    input_dir = os.path.join(bazel_bin, 'external', 'envoy_api', rule_dir, 'linux_amd64_stripped',
-                             proto + '%', IMPORT_BASE, rule_dir)
+    input_dir = os.path.join(bazel_bin, 'external', 'envoy_api', rule_dir, proto + '_', IMPORT_BASE,
```

Comment:
When I tried this locally, it could find the file in the old location. Was that some stale cache issue?

Reply:
I guess so, the a clean run result: https://github.com/lizan/go-control-plane/tree/11627